components/outerTemplate/src/components/footerComponent.js

```diff
@@ -9,33 +9,32 @@ function createPathToImage(src) {
 }
 // Create path to image end
 
+// Create foot content start
+function createFootItem(footContent) {
+  return footContent.map((cont) => {
+    return `<td align="center" width="24" style="padding: 0px 5px;" ng-show="slink.link != ''">
+    <a href="${cont.href}" target="_self">
+    <img width="24" alt="${cont.alt}" 
+    src="${createPathToImage(cont.src)}" 
+    style="display: block; border="0">
+    </a>
+  </td>`;
+  });
+}
+// Create foot content end
+
+// Create foot content start
+function createFootContent(footContent) {
+  return `<table role="presentation" cellpadding="0" cellspacing="0" border="0" align="left">
+  <tbody><tr>${createFootItem(footContent)}</tr></tbody>
+</table>`;
+}
+// Create foot content end
+
 export default function (footContent) {
   if (footContent == "") {
```

Comment:
as we removed everything from the middle of this function we probably can move if statement into `createFootContent` and get rid of this function